src/plugins/intel_cpu/src/emitters/snippets/x64/jit_snippets_emitters.cpp

```diff
@@ -120,7 +120,7 @@ void jit_scalar_emitter::emit_impl(const std::vector<size_t>& in, const std::vec
 }
 
 template <cpu_isa_t isa>
-void jit_scalar_emitter::emit_isa(const std::vector<size_t>& in, const std::vector<size_t>& out) const {
+void jit_scalar_emitter::emit_isa(const std::vector<size_t>& /*in*/, const std::vector<size_t>& out) const {
```

Comment:
The same thing. Why you decided to comment out the variable?

Reply:
There is a bug in gcc that was fixed in trunk only in 2020. The point is that `[[maybe_unused]]` is ignored in explicitly instantiated templates. You can see it here: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=90894